docs/guides/mirror_management/add_mirror_manager.md

```diff
@@ -40,41 +40,43 @@ Here are some crontab examples for you:
+You can also use and modify our example script implementing locking and full sync if required. It can be found at <https://github.com/rocky-linux/rocky-tools/blob/main/mirror/mirrorsync.sh>.
 
 After your first complete synchronization check that everything is fine with your mirror. Most importantly check all files and dirs got synchronized, your cron job is working properly and your mirror is reachable from the public Internet. Double check your firewall rules! To avoid any problems do not enforce http to https redirection.
 
-If you have any questions setting up your mirror join https://chat.rockylinux.org/rocky-linux/channels/infrastructure
+If you have any questions setting up your mirror join <https://chat.rockylinux.org/rocky-linux/channels/infrastructure>
 
 When you are done head over to the next section and propose your mirror to become public!
 
 ## What You Need
-* An account on https://accounts.rockylinux.org/
+
+- An account on <https://accounts.rockylinux.org/>
 
 ## Creating a site
 
 Rocky uses Fedora's Mirror Manager for organizing community mirrors.
 
-Access Rocky's Mirror Manager here: https://mirrors.rockylinux.org/mirrormanager/
+Access Rocky's Mirror Manager here: <https://mirrors.rockylinux.org/mirrormanager/>
 
 After a successful login, your profile will be on the top right. Select the drop down then click "My sites".
 
 A new page will load listing all sites under the account. The first time it will be empty. Click "Register a new site".
 
 A new page will load with an important Export Compliance statement to read. Then fill out the following information:
 
-* "Site Name"
-* "Site Password" - used by `report_mirrors` script, you make this anything you want
-* "Organization URL" - Company/School/Organization URL e.g. https://rockylinux.org/
-* "Private" - Checking this box hides your mirror from public use.
-* "User active" - Uncheck this box to temporarily disable this site, it will be removed from public listings.
-* "All sites can pull from me?" - Enable all mirror sites to pull from me without explicitly adding them to my list.
-* "Comments for downstream siteadmins. Please include your synchronization source here to avoid dependency loops."
+- "Site Name"
+- "Site Password" - used by `report_mirrors` script, you make this anything you want
+- "Organization URL" - Company/School/Organization URL e.g. <https://rockylinux.org/>
+- "Private" - Checking this box hides your mirror from public use.
+- "User active" - Uncheck this box to temporarily disable this site, it will be removed from public listings.
+- "All sites can pull from me?" - Enable all mirror sites to pull from me without explicitly adding them to my list.
+- "Comments for downstream siteadmins. Please include your synchronization source here to avoid dependency loops."
 
```

Comment:
@nishaaaaaant I believe @alemorvan has mentioned this before, but bullet items are NOT in error when prefixed with an *. A bullet can be either a * or -. Mixing those in the same bullet sequence is a markdown no-no, but either is acceptable as far as Markdown is concerned.